pandas/core/frame.py

```diff
@@ -6094,7 +6118,7 @@ def _arith_op(left, right):
         new_data = self._dispatch_frame_op(other, _arith_op)
         return new_data
 
-    def _construct_result(self, result) -> DataFrame:
```

Comment:
why was this deleted? self._constructor is typed as `Type[DataFrame]`. while not 100% correct, this should not be giving mypy errors.

Reply:
I think by mistake, reverting